### DISCUSSION TITLE
Bring back "Hit Animations" toggle in editor

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18285] [Ideas] [answered]
on #18239;
yes, toggleable please. its unusable. for me

**Comment by bdach** [comment]
first off, use actual words please, i can barely make out that you apparently want https://github.com/ppy/osu/pull/18239 reverted and want the hit animations toggle back.

secondly, we're not going to revert any change just because one person says it's "unusable". unusable why? what is the problem? what are you looking for? why would a revert fix your issues?

**Reply by Axilotl17** [reply]
> use actual words please

fair enough, i'm sorry that that wasn't clear.


> just because one person says it's "unusable".

it does say "feedback appreciated", although my feedback was poor. 


> unusable why? what is the problem?

It makes it really hard to see whats going on in the editor, it blocks overlapping notes, and it generally crowds the space. i personally cannot tell what's going on with those in the editor. toggleable, opacity, or even making it a kind of timer thing, like, a reverse approach circle? (or an approach circle that continues inside the circle or something) so you can see the order they went in after they were hit, in addition to showing their location. this would actually be more useful then what is currently happening. 


sorry for crappy original post, thats on me. :(

**Reply by peppy** [reply]
out of curiosity, have you ever used the osu(stable) editor?

**Reply by Axilotl17** [reply]
i have, I didn't like how the sliders worked at all, and it didn't run the best on linux, so i stopped using it.

**Comment by Pennek** [comment]
As a [ranked mapper](https://osu.ppy.sh/users/3723742) who maps low approach rate maps I can second this. Having visuals match gameplay is crucial to making readable maps. For most mappers it's a preference as maps around `AR9-10` aren't affected much, but it really affects more visually dense maps to the point of making things unreadable. This becomes worse as approach rate goes lower. It is especially concerning when there are a lot of overlapping circles.

Here are a few examples from my own maps:
AR8 on [this map](https://osu.ppy.sh/beatmapsets/1799229):
Lazer: https://youtu.be/mp66MJAEMNI
Stable: https://youtu.be/bnL-RLDr724

AR6.8 on [this map](https://osu.ppy.sh/beatmapsets/993369#osu/2077273): 
Lazer: https://youtu.be/U_LmN9h12cc
Stable: https://youtu.be/K8lHzamVkU8

AR4 on a map I tried to continue in Lazer:
Lazer: https://youtu.be/uyju80QdwoI
Stable: https://youtu.be/xpn5gzPmIDo

You can probably see where I am going with this, but I wouldn't even consider mapping anything below AR4 without a toggle as it just doesn't become feasible to read while mapping.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/21173

**Reply by Pennek** [reply]
Thank you!
